modules/nw-ipfailover-configuration.adoc

```diff
@@ -0,0 +1,179 @@
+          value: "false"
+        #- name: OPENSHIFT_HA_UNICAST_PEERS
+          #value: "10.0.148.40,10.0.160.234,10.0.199.110"
+        - name: OPENSHIFT_HA_IPTABLES_CHAIN <7>
+          value: "INPUT"
+        #- name: OPENSHIFT_HA_NOTIFY_SCRIPT <8>
+        #  value: /etc/keepalive/mynotifyscript.sh
+        - name: OPENSHIFT_HA_CHECK_SCRIPT <9>
+          value: "/etc/keepalive/mycheckscript.sh"
+        - name: OPENSHIFT_HA_PREEMPTION <10>
+          value: "preempt_delay 300"
+        - name: OPENSHIFT_HA_CHECK_INTERVAL <11>
+          value: "2"
+        livenessProbe:
+          initialDelaySeconds: 10
+          exec:
+            command:
+            - pgrep
+            - keepalived
+      volumes:
+      - name: lib-modules
+        hostPath:
+          path: /lib/modules
+      - name: host-slash
+        hostPath:
+          path: /
+      - name: etc-sysconfig
+        hostPath:
+          path: /etc/sysconfig
+      # config-volume contains the check script
+      # created with `oc create configmap keepalived-checkscript --from-file=mycheckscript.sh`
+      - configMap:
+          defaultMode: 0755
+          name: keepalived-checkscript
+        name: config-volume
+      imagePullSecrets:
+        - name: openshift-pull-secret <12>
+----
+<1> The list of IP address ranges to replicate. This must be provided. For example, `1.2.3.4-6,1.2.3.9`.
+<2> The number of groups to create for VRRP. If not set, a group is created for each virtual IP range specified with the `OPENSHIFT_HA_VIP_GROUPS` variable.
+<3> The interface name for IP failover uses to send VRRP traffic. By default, `eth0` is used.
+<4> The IP failover pod tries to open a TCP connection to this port on each VIP. If connection is established, the service is considered to be running. If this port is set to `0`, the test always passes. The default value is `80`.
+<5> The offset value used to set the virtual router IDs. Using different offset values allows multiple IP failover configurations to exist within the same cluster. The default offset is `0`, and the allowed range is `0` through `255`.
+<6> The number of replicas to create. This must match `spec.replicas` value in IP failover deployment configuration. The default value is `2`.
+<7> The name of the `iptables` chain to automatically add an `iptables` rule to allow the VRRP traffic on. If the value is not set, an `iptables` rule is not added. If the chain does not exist, it is not created, and Keepalived operates in unicast mode. The default is `INPUT`.
+<8> The full path name in the pod file system of a script that is run whenever the state changes.
+<9> The full path name in the pod file system of a script that is periodically run to verify the application is operating.
+<10> The strategy for handling a new higher priority host. The default value is `preempt_delay 300`, which causes a Keepalived instance to take over a VIP after 5 minutes if a lower-priority priority master is holding the VIP.
```

Comment:
There is an extra "priority" in the sentence of "..., which causes a Keepalived instance to take over a VIP after 5 minutes if a lower-priority priority master is holding the VIP" that needs to be removed.

Reply:
Thanks! Removed.